src/components/DrsStarterKit.js

```diff
@@ -0,0 +1,30 @@
+import '@fontsource/roboto';
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  useRouteMatch
+} from "react-router-dom";
+import DrsObject from './DrsObject'
+
+function DrsStarterKit() {
```

Comment:
let's rename this file. The `StarterKit` of the file name doesn't accurately describe what this page does. I would call this an **index** page since it lists the `DrsObjects`. So Maybe `DrsIndex.js`. Also, add a subdirectory to the component library for each standard, i.e.:

- `src/components/pages/drs/DrsIndex.js`

that way all pages associated with one API are grouped